.vale/styles/RedHat/Spelling.yml

```diff
@@ -151,6 +151,8 @@ filters:
   - Che-Theia
   - Ciphertext
   - Civetweb
+  - ClassLoader
```

Comment:
For this one, I just want to mention that in Java world we are seeing various notations:
- ClassLoader (which initially was to refer to the classname https://docs.oracle.com/javase/7/docs/api/java/lang/ClassLoader.html )
- class loader or Class Loader (with a space, maybe the most used notation in Java specification, 59 times on the [page](https://docs.oracle.com/javase/specs/jvms/se7/html/jvms-5.html) describing it)
- classloader or Classloader (without space, for instance used in red hat doc [here](https://access.redhat.com/documentation/en-us/jboss_enterprise_application_platform/5/html/microcontainer_user_guide/sect-jboss_microcontainer_user_guide-using_services-dynamic_classloading) and in Java specification [here](https://docs.oracle.com/javase/specs/jvms/se7/html/jvms-5.html#jvms-5.3) and in [title of wikipedia article](https://en.wikipedia.org/wiki/Java_Classloader))

Reply:
@apupier - Thanks for flagging this. Wondering if we need to add exceptions for all of the above variations. What do you think?

Reply:
Given how much the different variations are spread, I think it is more consensual to allow all of these variations. So I vote for allowing all of them. But I'm not a Technical Writer so take my words with a grain of salt.

Reply:
I changed the spelling yml instance to _classloader_ to prevent a spelling error on any of these: classloader/Classloader/ClassLoader.

As _ClassLoader_ is correct, when used to refer specifically to the Java class name and _classloader_ is also a correct form, I added ClassLoader to the Headings.yml.  
The other variation, _class loader_ doesn't generate a vale warning.

If there's any further discussion needed, we can bring this to the CCS style council for advice.

Thanks @apupier :+1: